mno_analysis_tools/fetch_archives.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+
+from core_data_modules.logging import Logger
+from rapid_pro_tools import RapidProClient
+
+log = Logger(__name__)
+log.set_project_name("FetchRawMessages")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Fetches archives from Rapid Pro instance")
+    parser.add_argument("domain", metavar="domain",
+        help="Domain that the instance of Rapid Pro is running on",
+    )
+    parser.add_argument("token", metavar="token",
+        help="Token for authenticating to the instance",
+    )
+    parser.add_argument("output_file", metavar="file", type=argparse.FileType(mode="w"),
+        help="File to write the raw data downloaded as json.",
+    )
+
+    args = parser.parse_args()
+  
+    source_domain = args.domain
+    source_token = args.token
+    output_file = args.output_file
+
+    source_instance = RapidProClient(source_domain, source_token)
+
+    log.info("fetch raw messages...")
+    source_instance.get_raw_messages(raw_export_log_file=output_file)
```

Comment:
Please serialize the data yourself rather than using the log_file argument. 

The log file's intended use is for maintaining a record of fetches e.g. when we do incremental fetching, we can append new data to this file then use it to debug errors in the incremental data. It's format happens to coincide with the one that you want, but may not in future and would cause you painful upgrade issues.

On re-reading the documentation for that argument, I realise that may not have been clear. I will file an issue to improve it/make that option private.